javascript/ql/src/semmle/javascript/TypeScript.qll

```diff
@@ -1550,14 +1550,20 @@ class ReferenceImport extends LineComment {
   string getAttributeName() { result = attribute }
 
   /**
+   * DEPRECATED. This is no longer supported.
```

Comment:
OOI, why do we need to drop this?

Reply:
It caused location info to be computed for all line comments.

Reply:
Ugh, yeah, I can see that being a problem.